tests/integration/targets/latest/tasks/controller.yml

```diff
@@ -0,0 +1,114 @@
+---
+- debug:
+    msg: "{{ nautobot_version }}"
+
```

Comment:
```suggestion
```

Reply:
I think it is alright to leave the debug message in here to print out for CI.

Reply:
I don't want to start a copy/pasta pattern for future modules. Can you add it to the `main.yml` instead and update the message to something more human readable?

Reply:
If you want, we could also move all of the existing conditional blocks out of the individual test files and into `main.yml`. It would then make sense to have a debug message there like "Running 2.2+ tests" and "Skipping 2.2+ tests because version is {{nautobot_version}}".

Reply:
I am approving this PR for now, but I'll want to come back and revisit this later if we need to continue to need to have tests conditionally run based on the version.